action.yml

```diff
@@ -0,0 +1,67 @@
+name: "ssm-put-parameter"
+description: "Puts parameter to AWS Parameter Store using AWS CLI"
+inputs:
+  name:
+    description: "Name of the parameter"
+    required: true
+
+  value:
+    description: "Value of the parameter"
+    required: true
+
+  description:
+    description: "Description of the parameter"
+    required: false
+    default: ""
+
+  type:
+    description: "Type of the parameter: [String | SecureString | StringList]"
+    required: false
+    default: "String"
+
+  overwrite:
+    description: "If needed to overwrite the parameter: ['overwrite' | 'no-overwrite']"
+    required: false
+    default: "no-overwrite"
```

Comment:
I'd assume that `overwrite` should be the default. If you run this twice it'll fail the second time right?

Reply:
changed default value to `overwrite`

Reply:
Needs updating in the readme too!

Reply:
oh, my bad, updated readme too